src/common_tests.py

```diff
@@ -57,21 +59,26 @@ def test_records_from_file_stream(self):
 
             self.assertEqual(expected, actual)
 
+    @nottest
```

Comment:
Are these marked as nottest only because they fail on windows?

In that case, do you want to assign an attribute to them and add a note in the README about testing on windows?

https://nose.readthedocs.io/en/latest/plugins/attrib.html

e.g.
`@attr('fails_on_windows')`

and then

`nosetests -a '!fails_on_windows``

Reply:
Nice, I didn't know about `@attr` , reverted the tests to their previous state and added attr.